packages/react/src/components/password-creation-input/password-creation.test.tsx

```diff
@@ -0,0 +1,139 @@
+import { shallow } from 'enzyme';
+import { PasswordCreationInput } from './password-creation-input';
+import { mountWithProviders } from '../../test-utils/renderer';
+import { getByTestId } from '../../test-utils/enzyme-selectors';
+import { Translations } from '../../i18n/translations';
+
+describe('PasswordCreationInput', () => {
+    const onChange = jest.fn();
+
+    test('sets password type to password when not showing password', () => {
+        const wrapper = shallow(
+            <PasswordCreationInput onChange={onChange} />,
+        );
+
+        expect(getByTestId(wrapper, 'password-input').prop('type')).toBe('password');
+    });
+
+    test('calls onChange with password and isValid to false when a custom validation is false', () => {
+        const wrapper = mountWithProviders(
+            <PasswordCreationInput
+                validations={[
+                    {
+                        label: 'contains numbers',
+                        isValid: (password) => /\d/.test(password),
+                    },
+                ]}
+                onChange={onChange}
+            />,
+        );
+        const password = 'abc';
+        const event = { target: { name: 'password', value: password } };
+
+        getByTestId(wrapper, 'password-input').prop('onChange')(event);
```

Comment:
Tu pourrais utiliser `.simulate`
```suggestion
        getByTestId(wrapper, 'password-input').simulate('change',  { target: { name: 'password', value: password } });
```

Reply:
j'avais essayé et ça fonctionne pas avec simulate. je sais pas trop pourquoi. won't fix 
<img width="1047" alt="Capture d’écran, le 2022-01-07 à 13 23 28" src="https://user-images.githubusercontent.com/19400226/148589470-5c4dcb56-bd2e-4666-8a0e-823e68e9201f.png">